src/upcoming-components/Drawer/Drawer.tsx

```diff
@@ -225,7 +226,6 @@ export const Drawer = ({
             className={styles['drawer__window']}
             ref={windowRef}
             role="dialog"
-            tabIndex={0} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
           >
```

Comment:
error: `tabIndex` should only be declared on interactive elements.

because <article> is static element, so removed tabIndex, added it to parent interactive element